<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportContractsRegistrySetupDialog</class>
 <widget class="QDialog" name="ReportContractsRegistrySetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">    
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtBegDate"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата начала периода</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="CTimeEdit" name="edtBegTime"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblBegTime">
     <property name="text">
      <string>Время начала периода</string>
     </property>
    </widget>
   </item>
  <item row="3" column="1">
    <widget class="CDateEdit" name="edtEndDate"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата окончания периода</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="CTimeEdit" name="edtEndTime"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblEndTime">
     <property name="text">
      <string>Время окончания периода</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblContract">
     <property name="text">
      <string>Договор</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="CContractComboBox" name="cmbContract"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Исполнитель</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="CPersonComboBox" name="cmbPerson"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblFinance">
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="CRBComboBox" name="cmbFinance"/>
   </item> 
   <item row="9" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>Тип обращения</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
    <item row="10" column="0">
    <widget class="QLabel" name="lblVisitPayStatus">
     <property name="text">
      <string>Флаг Финансирования</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="cmbVisitPayStatus">
      <property name="minimumSize">
        <size>
         <width>252</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>не задано</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>не выставлено</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>выставлено</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>отказано</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>оплачено</string>
      </property>
    </item>
    </widget>
   </item>
    <item row="12" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item> 
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CTimeEdit</class>
   <extends>QTimeEdit</extends>
   <header>library/TimeEdit</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBox</header>
  </customwidget>
  <customwidget>
   <class>CContractComboBox</class>
   <extends>QComboBox</extends>
   <header>Accounting/ContractComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportContractsRegistrySetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>418</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportContractsRegistrySetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>650</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
